releases/0.8.0/index.md

```diff
@@ -18,14 +18,16 @@ custom_css: releases
 *(These are changes to ontologies, classes or properties in the preexisting ontology that make the new release non-backward-compatible.)*
 
 * Restructured repository by moving previously prefixed uco-* ontology files under new directory /ontology/* ([*Change Proposal 56*](https://drive.google.com/file/d/1PCjdCGw7wgFnPfbsCn0Fdvdnq9NDW6yA/view))
+* Remove ActionReferenceFacet and move properties into action:Action ([*Change Proposal 89*](https://drive.google.com/file/d/1njOUmpNNCTolnGkXoKv5ext-ZK18108u/view))
```

Comment:
The document attached is missing the implementation completion date, you may want to update this to the date of the PR merge then re-upload the PDF. I am unsure if the link will change.